### PR TITLE
feat(Select): added errorMessage, errorPlacement and validationType props

### LINKED ISSUE #1273: Select: add error view like in TextInput component
- deprecate property [error](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/types.ts#L85)
- add property [errorMessage](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/types.ts#L39)
- add property [errorPlacement](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/types.ts#L39)
- add property [validationState](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/types.ts#L43C9-L43C24)

### PR DESCRIPTION
**Pull request by kkirik**
Closes #1273

### CODE REVIEW

File: src/components/Select/components/SelectControl/SelectControl.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 
-import {ChevronDown} from '@gravity-ui/icons';
+import {ChevronDown, TriangleExclamation} from '@gravity-ui/icons';
 import isEmpty from 'lodash/isEmpty';
+import type {CnMods} from 'src/components/utils/cn';
```
Comment (korvin89) [inline src/components/Select/components/SelectControl/SelectControl.tsx:5]: We don't support aliases. Btw, CI tests failed because of it

Comment (kkirik) [reply src/components/Select/components/SelectControl/SelectControl.tsx:5]: Thanks. I'll check

**Comment by korvin89** [comment]
One more thing: could u add some tests for this features? U can get tests from [here](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L91). And add QA constant [here](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/constants.ts#L34) please. And there is some different behaviour regarding TextInput (in [this](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L98) test case, but it has a confusing name :))

**Comment by kkirik** [comment]
> One more thing: could u add some tests for this features? U can get tests from [here](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L91). And add QA constant [here](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/constants.ts#L34) please. And there is some different behaviour regarding TextInput (in [this](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L98) test case, but it has a confusing name :))

So, I added some tests for error feature also I added an aria-label attribute for getting internal error icon by i11y attributes.

**Review by korvin89 — APPROVED** [review]

**Pull request merged**